multi/imxrt-multi/uart.c

```diff
@@ -67,12 +69,21 @@ typedef struct uart_s {
 	size_t txFifoSz;
 
 	libtty_common_t tty_common;
+
+	/* fifo between irq and thread */
+	lf_fifo_t rxFifoCtx;
+	uint8_t rxFifoData[UART_RXFIFOSIZE];
+
+	/* statistics */
+	struct {
+		size_t hw_overrunCntr;
+		size_t sw_overrunCntr;
+	} stat;
```

Comment:
This struct seems write-only

Reply:
imxrt-multi does not have mechanisms for reading counters (yet), this structure is helpful for reading with a debugger. I can add a comment.